types/time_test.go

```diff
@@ -288,7 +288,6 @@ func TestDate(t *testing.T) {
 		"2019 01 02",
 		"2019A01A02",
 		"2019-01T02",
-		"2011-12-13 10:10T10",
```

Comment:
mysql 8.0.18 support this
mysql> SELECT CAST("2020-11-2 11:11T11" AS DATETIME);
+----------------------------------------+
| CAST("2020-11-2 11:11T11" AS DATETIME) |
+----------------------------------------+
| 2020-11-02 11:11:00                    |
+----------------------------------------+
1 row in set, 1 warning (0.00 sec)

mysql> select version();
+-----------+
| version() |
+-----------+
| 8.0.18    |
+-----------+
1 row in set (0.02 sec)